tests/exporters/exporters_utils.py

```diff
@@ -132,6 +132,7 @@
         "hf-internal-testing/tiny-random-vision_perceiver_conv": ["image-classification"],
     },
     "phi": "echarlaix/tiny-random-PhiForCausalLM",
+    "phi3": "katuni4ka/tiny-random-phi3",
```

Comment:
what do you think about also adding a test to [test_modeling.py](https://github.com/huggingface/optimum/blob/e0f58121140ce4baa01919ad70a6c13e936f7605/tests/onnxruntime/test_modeling.py#L2248) using this model

Reply:
Added!